### ISSUE TITLE
Support for themes?

### ISSUE THREAD

**Issue by tavuntu** [cebor/angular-highcharts#59]
Does this plugin have theme support? it would be nice to have a way of doing somethins like [this](https://github.com/highcharts/highcharts/blob/master/js/themes/dark-blue.js) directly with Angular.

**Comment by cebor** [comment]
@tavuntu yes, the following should work.

```typescript
import { Highcharts } from 'angular-highcharts';

// Apply the theme
Highcharts.setOptions(Highcharts.theme);
```

**Comment by tavuntu** [comment]
The object HIghcharts is undefined when importing it, so it throws an error when trying to use .theme or .setOptions(). I've added the ChartModule un app.moduel.ts, maybe I'm missing something else?

**Comment by cebor** [comment]
To resolve this issue i need more code to dig into this. Can you setup a minimal repo or Plunkr?

**Comment by tavuntu** [comment]
here you go, I'm using npm 5.4.0 and the latest version of angular-highcharts
https://github.com/tavuntu/general-stuff

**Comment by faridv** [comment]
I get `error TS2339: Property 'theme' does not exist on type 'Static'.`
I checked and I couldn't find `theme` in Highchart interface. Am I doing something wrong or this project doesn't support themes yet?

**Comment by SayakMukhopadhyay** [comment]
Was this issue resolved? Is there any workaround?

**Comment by tavuntu** [comment]
@SayakMukhopadhyay  I haven't tested this since then (I switched to Chart.js), are seeing the issue?

**Comment by SayakMukhopadhyay** [comment]
Yeah, I am. I am planning on the only workaround of loading the theme js in the header for now. Using `setOptions` is more flexible though.

**Comment by SayakMukhopadhyay** [comment]
UPDATE:
The procedure mention in https://github.com/cebor/angular-highcharts/issues/59#issuecomment-327407520 is working in Angular 5. Tried and tested!

**Comment by dereklin** [comment]
Highcharts.setOptions(Highcharts.theme); seems to affect the new charts after it.

What would the easiest way to refresh the charts so that the new theme is applied immediately to existing charts?

**Comment by dereklin** [comment]
Interesting, so you are basically recreating the chart when switching themes right?

```
this.themeService.theme$.subscribe(theme => {
            this.chart = new Chart(this.options);
        });
```

By doing 

```
this.linkRef.onload = () => this.themeService.themeLoaded();
``` 
and 
```
```
this.linkRef.href = this.themeService.getTheme().href;
```

You are loading the style sheet related to the theme dynamically and also triggering the theme change, right?

**Comment by SayakMukhopadhyay** [comment]
Yup. Recreating the chart object is the only way to change the theme as far as what I have found.

**Comment by wiggitamoo** [comment]
I was able to get it working like this: 
```
this.chart = new Chart({...});
this.chart.theme = {...};
Highcharts.setOptions(this.chart.theme);
```
